aptos-move/framework/src/natives/cryptography/bulletproofs.rs

```diff
@@ -91,6 +104,54 @@ fn native_verify_range_proof(
     verify_range_proof(context, &comm_point, &pg, &proof_bytes[..], num_bits, dst)
 }
 
+fn native_verify_batch_range_proof(
```

Comment:
This needs to also charge a base cost (see the native for individual proof verification).

@zjma do you know why this is missing / what happened here

Reply:
the batch verification implementation is multivariate and non-linear. I suggested we use a value table for gas charging, so `BULLETPROOFS_BASE` is not needed.

Reply:
Can you double check with @vgao1996. I recall he always advises to have a base cost.

Reply:
I think @vgao1996's point was: every native needs to have a non-zero cost, but that's it.